includes/leads.js

```diff
@@ -0,0 +1,49 @@
+module.exports = (params) => {
+  return publish("salesforce_leads", {
+    description: "Base model for Salesforce Leads",
+    columns: {
+      lead_id: "Unique ID for the Lead object.",
+      email_hash: "Hashed Email address",
+      name_hash: "Hashed Name of Lead",
+      status: "Lead Status, usually from dropdown.",
+      lead_source: "Source of Lead, based on simple attribution models.",
+      is_converted: "Flag denoting whether lead was converted to a contact. BOOLEAN",
+      created_date: "Calendar date for when the lead was created, DATETIME",
+    },
+    assertions: {
+      uniqueKey: ['lead_id'],
+      nonNull: ['lead_id', 'email_hash']
+    },
+    ...params.defaultConfig
+  }).query(ctx => `
+
+  with base as (
+        select *
+        from ${ctx.ref(params.salesforceSchema, params.leadsTableName)}
+        where not isdeleted
```

Comment:
sure we want to do this here? Depends on what deletion means exactly in SF, but I can see situations where you may not want this filter to be applied

Reply:
I'm not sure about this though, as it's confusing in other ways

Reply:
It's quite common for Salesforce objects to be in a "soft-deleted" state - sales managers, for instance, usually have admin privileges are able to delete opportunity records. For this reason, it is best to ensure all delete records are filtered out at the source.